documentation/docs/p2p.mdx

```diff
@@ -42,116 +41,72 @@ deactivate Issuer
 
 Based on the diagram above the flow is as follows:
 1. Sender submits payment to its own wallet
-2. Issuer wallet gets Open Payment metadata
-3. Issuer wallet creates invoice on receivers wallet
+2. Issuer wallet creates invoice on receivers wallet
 3. Issuer pays invoice
 
 ## 1. Sender submits payment
 
-The sender presents the Payment Pointer of the Receiver to its Issuer to instruct it to make a Peer-to-Peer payment
+The sender presents its Issuer with the payment identifier (eg. Payment Pointer) of the Receiver and the amount they want to send. 
+This will instruct the Issuer to make a Peer-to-Peer payment.
 
-## 2. Discover Open Payments metadata
+## 2. Sending wallet creates invoice 
 
-Using the Payment Pointer provided, the Merchant will perform a GET against the correct url.
+Given the sender provided the Issuer wallet with the receiver's Payment Pointer of `$acquirer.wallet/bob`,
```

Comment:
Is the goal that for all fixed source amount / P2P payments, the sender would always create an invoice first? i.e., is the `/connections` resources on the account just for legacy reasons to support SPSP?